tools/packaging/packaging.psm1

```diff
@@ -30,226 +30,261 @@ function Start-PSPackage {
+            'nupkg' {
+                $Arguments = @{
+                    PackageNameSuffix = $NameSuffix
+                    PackageSourcePath = $Source
+                    PackageVersion = $Version
+                    PackageRuntime = $Runtime
+                    PackageConfiguration = $Configuration
+                    Force = $Force
+                }
+
+                if ($PSCmdlet.ShouldProcess("Create NuPkg Package")) {
+                    New-NugetPackage @Arguments
+                }
             }
+            'tar' {
+                $Arguments = @{
+                    PackageSourcePath = $Source
+                    Name = $Name
+                    Version = $Version
+                    Force = $Force
+                }
 
-            if ($PSCmdlet.ShouldProcess("Create tar.gz Package")) {
-                New-TarballPackage @Arguments
+                if ($PSCmdlet.ShouldProcess("Create tar.gz Package")) {
+                    New-TarballPackage @Arguments
+                }
             }
-        }
-        'deb' {
-            $Arguments = @{
-                Type = 'deb'
-                PackageSourcePath = $Source
-                Name = $Name
-                Version = $Version
-                Force = $Force
+            'deb' {
+                $Arguments = @{
+                    Type = 'deb'
+                    PackageSourcePath = $Source
+                    Name = $Name
+                    Version = $Version
+                    Force = $Force
+                }
+                foreach ($Distro in $Script:DebianDistributions) {
+                    $Arguments["Distribution"] = $Distro
+                    if ($PSCmdlet.ShouldProcess("Create DEB Package for $Distro")) {
+                        New-UnixPackage @Arguments
+                    }
+                }
             }
-            foreach ($Distro in $Script:DebianDistributions) {
-                $Arguments["Distribution"] = $Distro
-                if ($PSCmdlet.ShouldProcess("Create DEB Package for $Distro")) {
+            default {
+                $Arguments = @{
+                    Type = $_
+                    PackageSourcePath = $Source
+                    Name = $Name
+                    Version = $Version
+                    Force = $Force
+                }
+
+                if ($PSCmdlet.ShouldProcess("Create $_ Package")) {
                     New-UnixPackage @Arguments
                 }
             }
         }
-        default {
-            $Arguments = @{
-                Type = $_
-                PackageSourcePath = $Source
-                Name = $Name
-                Version = $Version
-                Force = $Force
-            }
 
-            if ($PSCmdlet.ShouldProcess("Create $_ Package")) {
-                New-UnixPackage @Arguments
-            }
+        if($IncludeSymbols.IsPresent)
+        {
+            Remove-Item -Path $Source -Recurse -Force -ErrorAction SilentlyContinue
```

Comment:
It would be great if you can put a comment here saying that "$Source is point to a temporary folder when -IncludeSymbols is present".